### DISCUSSION TITLE
"setCustomValidity in JavaScript." But what is the effect of that?

### DISCUSSION THREAD

**Discussion by batje** [twbs/bootstrap#36638] [Q&A]
The documentation about validation states:

"You may provide custom validity messages with setCustomValidity in JavaScript."

But all the examples have hardcoded <valid-feedback> or <invalid-feedback> sections.

Is there a way to use the original Validation messages with Bootstrap?

**Comment by miloskroulik** [comment]
It looks like this information should be moved to the https://getbootstrap.com/docs/5.3/forms/validation/#browser-defaults part, because it works fine then - not with the custom BS styles.
